### PR TITLE
refactor: use whiskers

### PR DESCRIPTION
**Pull request by thelooter**
Adds [Whiskers](https://github.com/catppuccin/toolbox/tree/main/whiskers), Catppuccin's in-house templating tool, to build the themes. Instead of editing the `flavor.css` files directly, edit  the `codemirror.tera` file and then run just build to build the output themes. You will need Whiskers (linked above) and optionally [Just](https://github.com/casey/just) installed.

### CODE REVIEW

File: codemirror.tera [hunk 1]
```diff
@@ -0,0 +1,117 @@
+---
+accent: mauve
```
Comment (uncenter) [inline codemirror.tera:2]:
```suggestion
```

File: codemirror.tera [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+  --atom: #{{ lavender.hex }};
+  --cursor: #{{ overlay1.hex }};
+  --keyword: #{{ red.hex }};
+  --operator: #{{ sky.hex }};
+  --number: #{{ peach.hex}};
+  --definition: #{{ blue.hex }};
+  --string: #{{ green.hex }};
+}
+
+.cm-s-ctp-{{ flavor.identifier }}.CodeMirror,
```
Comment (uncenter) [inline codemirror.tera:32]:
```suggestion
.cm-s-ctp-{{ flavor.identifier }}.CodeMirror,
```

This line is super repetitive. I would set it to a variable like `{% set prefix = ".cm-s-ctp-" ~ flavor.identifier %}`.

File: justfile [hunk 3]
```diff
@@ -0,0 +1,5 @@
+_default:
+  @just --list
+
+build:
+  whiskers codemirror.tera
```
Comment (uncenter) [inline justfile:5]: Newlines please!

**Review by uncenter — CHANGES_REQUESTED** [review]

**Comment by griimick** [comment]
Hey! Can you also add a section in README with instructions for setting up a dev environment to build the theme with the current changes? Sorry, I am not up to date with the whiskers setup but I would like to know how to build the theme with the new build system.

**Comment by uncenter** [comment]
> Hey! Can you also add a section in README with instructions for setting up a dev environment to build the theme with the current changes? Sorry, I am not up to date with the whiskers setup but I would like to know how to build the theme with the new build system.

I don't think we are opting to go with Whiskers for this port.

**Comment by uncenter** [comment]
I'm going to close this for now, @griimick I can direct you to our conversation in `#dev` if needed.

**Pull request closed without merging**